crypto/cms/cms_ec.c

```diff
@@ -24,23 +24,16 @@ static EVP_PKEY *pkey_type2param(int ptype, const void *pval,
     if (ptype == V_ASN1_SEQUENCE) {
         const ASN1_STRING *pstr = pval;
         const unsigned char *pm = pstr->data;
-        int pmlen = pstr->length;
+        size_t pmlen = (size_t)pstr->length;
```

Comment:
Can this ever be -1 - that would be bad :)

Reply:
If anyone assigns a negative number to the ASN1_STRING length anywhere, things will most likely go _kaboom_.  The assumption is that it's >=0